addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -38,6 +38,10 @@ html, body {
     }
 }
 
+.disable_pointer_events {
+    pointer-events: none;
+}
```

Comment:
ẁeb_editor.common.scss is the file which contains the style for the visitor of an edited content. Probably use `wysiwyg.scss` or something like that.